### PR TITLE
i18n: Update RTLCSS to 2.0.5

### PR DESCRIPTION
**Pull request by yoavf**
The most significant change is the support for `calc`, but settings have also been changed/renamed.

### CODE REVIEW

**Comment by yoavf** [comment]
Fixes #3451 , and improves overall RTL css support.
The changes to the stats post trends are necessary to prevent the arrows from being flipped.

File: .rtlcssrc [hunk 1]
```diff
@@ -1,10 +1,6 @@
...
-    "options": {
-        "preserveComments": true,
-        "preserveDirectives": false,
-        "swapLeftRightInUrl": true,
-        "swapLtrRtlInUrl": true,
-        "swapWestEastInUrl": false,
-        "autoRename": false
-    }
+	"options": {
+		"processUrls": { "atrule": true, "decl": false }
```
Comment (gwwar) [inline .rtlcssrc]: What type of node types is this targeting? I couldn't quite grok what the doc was describing https://github.com/MohammadYounes/rtlcss#options-object

**Comment by gwwar** [comment]
When testing with the HE locale, it seems like this branch is left-to-right, but WordPress.com is right-to-left. Are there other test steps I'm missing? 

left-to-right-branch

File: .rtlcssrc (hunk 1 above)
Comment (yoavf) [inline .rtlcssrc]: 'atrule' affects URLs in at-rules such as
`@import url("example-ltr.css");`

'decl' affects urls in declarations:
`background: url( 'something-rtl.png');`

**Comment by yoavf** [comment]
@gwwar thanks for the review. For local testing you need to set `rtl` to `true` in `config/development.json`

File: .rtlcssrc (hunk 1 above)
Comment (gwwar) [inline .rtlcssrc]: :) Thanks for the clarification!

**Comment by gwwar** [comment]
👍 Confirmed the issue in prod, verified fixed on the branch. Shrinkwrap changes look good.
fixed

**Pull request merged**